src/openms/source/ANALYSIS/XLMS/OpenPepXLLFAlgorithm.cpp

```diff
@@ -997,5 +997,6 @@ using namespace OpenMS;
     pep_indexing.run(fasta_db, protein_ids, peptide_ids);
 
     OPXLHelper::addProteinPositionMetaValues(peptide_ids);
+    OPXLHelper::addXLTargetDecoyMV(peptide_ids);
```

Comment:
no addBetaAccessions here?

Reply:
right now addProteinPositionMetaValues also adds beta accessions. If I remove that functionality from there and call both functions, I will have the same result but will have to loop over all IDs twice. Not sure if it is worth it currently.

Reply:
Hmm sounds a bit confusing. Is this properly documented?